uint8 agent_number
geographic_msgs/GeoPoint source
float32 angle
float32 wind_speed
float32 diff_y
float32 diff_z
float32 release_rate
